src/test/java/de/rabea/game/BoardTest.java

```diff
@@ -20,176 +20,298 @@ public void setup() {
     }
 
     @Test
-    public void placesAMark() {
-        Board board = new Board();
-        Mark[] cells = {EMPTY, X, EMPTY,
-                        EMPTY, EMPTY, EMPTY,
-                        EMPTY, EMPTY, EMPTY};
-        board.placeMark(1, X);
-        assertArrayEquals(cells, board.cells());
-    }
-
-    @Test
-    public void boardIsEmpty() {
-        Board emptyBoard = new Board();
+    public void boardIsEmpty3x3() {
+        Board emptyBoard = new Board(3);
         assertFalse(emptyBoard.isFull());
     }
 
     @Test
-    public void boardIsFull() {
-        Board fullBoard = fullBoardNoWinner();
+    public void boardIsFull3x3() {
+        Board fullBoard = full3x3BoardNoWinner();
         assertTrue(fullBoard.isFull());
     }
 
     @Test
-    public void isChosenPositionStillFree() {
-        Board emptyBoard = new Board();
+    public void isChosenPositionStillFreeOn3x3() {
+        Board emptyBoard = new Board(3);
         assertEquals(true, emptyBoard.isPositionAvailable(1));
     }
 
     @Test
-    public void markedPositionIsNoLongerAvailable() {
-        Board board = new Board();
-        board.placeMark(1, X);
-        assertEquals(false, board.isPositionAvailable(1));
+    public void markedPositionIsNoLongerAvailableOn3x3() {
+        Board board = new Board(3);
+        board.placeMarkOnExistingBoard(1, X);
+        assertFalse(board.isPositionAvailable(1));
+    }
+
+    @Test
+    public void markedPositionIsNoLongerAvailableOn4x4() {
+        Board board = new Board(4);
+        board.placeMarkOnExistingBoard(1, X);
+        assertFalse(board.isPositionAvailable(1));
     }
 
     @Test
-    public void itKnowsWhenAPositionIsUnavailable() {
-        Board board = new Board();
+    public void knowsWhenPositionIsUnavailableOn3x3() {
+        Board board = new Board(3);
         int positionNotOnBoard = 9;
         assertFalse(board.isPositionAvailable(positionNotOnBoard));
     }
 
     @Test
-    public void returnsAllEmptyPositions() {
-        Board board = new Board();
-        board.placeMark(1, X);
-        board.placeMark(2, O);
-        board.placeMark(3, X);
-        board.placeMark(6, O);
+    public void knowsWhenPositionIsAvailableOn4x4() {
+        Board board = new Board(4);
+        int positionOnBoard = 9;
+        assertTrue(board.isPositionAvailable(positionOnBoard));
+    }
+
+    @Test
+    public void listsAllEmptyPositionsFor3x3() {
+        Board board = new Board(3);
+        board.placeMarkOnExistingBoard(1, X);
+        board.placeMarkOnExistingBoard(2, O);
+        board.placeMarkOnExistingBoard(3, X);
+        board.placeMarkOnExistingBoard(6, O);
         List<Integer> emptyCells = Arrays.asList(0, 4, 5, 7, 8);
```

Comment:
`emptyCells` can be inlined and `Arrays.asList(...)` can be statically imported to improve readability.